frontend/src/services/api.ts

```diff
@@ -5,12 +5,10 @@ const WAIT_FOR_AUTH_DELAY_MS = 500;
 
 export async function request(
   url: string,
-  optionsIn: RequestInit = {},
+  options: RequestInit = {},
```

Comment:
You've changed the name of the parameter but not its name when its used.

Reply:
Whoops 😅 thanks